pandas/core/arrays/base.py

```diff
@@ -216,6 +216,27 @@ def isna(self):
         """
         raise AbstractMethodError(self)
 
+    def argsort(self, axis=-1, kind='quicksort', order=None):
+        """Returns the indices that would sort this array.
+
+        Parameters
+        ----------
+        axis : int or None, optional
+            Axis along which to sort. ExtensionArrays are 1-dimensional,
+            so this is only included for compatibility with NumPy.
+        kind : {'quicksort', 'mergesort', 'heapsort'}, optional
+            Sorting algorithm.
+        order : str or list of str, optional
+            Included for NumPy compatibility.
```

Comment:
Is this compatibility needed because in the code we use `np.argsort(values)` which passes those keywords to the method? 
(it is a bit unfortunate ..)

Reply:
It's not necessary, and I can remove it.

I see now that Categorical.argsort has a different signature. I suppose we should match that.